uint8   joint_num
---
sensor_msgs/JointState  joint_state